src/Components/Color/ColorCard.jsx

```diff
@@ -0,0 +1,59 @@
+import { useState } from "react";
+import "./Color.css";
+import ColorForm from "./ColorForm";
+
+/* functional component named ColorCard */
+/* receives a single prop called color, which is an object */
+
+export default function ColorCard({ color, onDelete, onUpdate }) {
```

Comment:
You are passing the `onUpdate` function here to your `ColorCard` component. But you actually do not have an `onUpdate` function in your `App` component and you do not pass it from there to your `ColorCard` component. So you first need to write the function, which takes care of the update and then pass it from your `App` component to the `ColorCard` component.

Reply:
thanks for your advice!:) I still have difficulties with when to commit and when to open the pull-request. Currently I try to open the pull-request once I am working on one issue and commit as soon as I finish one little peace (like a function e.g.). So that is why I run into not yet finished parts.